src/java.desktop/windows/classes/sun/awt/windows/WClipboard.java

```diff
@@ -83,13 +83,7 @@ protected void setContentsNative(Transferable contents) {
                         translateTransferable(contents, flavor, format);
                     publishClipboardData(format, bytes);
                 } catch (IOException e) {
-                    // Fix 4696186: don't print exception if data with
-                    // javaJVMLocalObjectMimeType failed to serialize.
-                    // May remove this if-check when 5078787 is fixed.
-                    if (!(flavor.isMimeTypeEqual(DataFlavor.javaJVMLocalObjectMimeType) &&
```

Comment:
https://bugs.openjdk.java.net/browse/JDK-5078787 is not fixed .. so .. ?

Reply:
I think printing the exception here isn't necessary because the code loops through all possible formats and checks if encoding is possible. If encoding isn't possible then it should skip the format and not print an error.